packages/ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@campgladiator/ui",
-  "version": "1.1.42",
+  "version": "1.1.43",
```

Comment:
@marinagustint Your changes in this PR represent a major version update to CGUI since you are now requiring new classes to be added in order to access the same styles. 

This should only require a minor or patch update that doesn't require any changes to how those styles are accessed (meaning no additional class names should be required).

Reply:
@websmith - Seems like the current CGUI library for headings is conflicting with the new font sizes and also in the old CGUI library things were differentiate accurately for "gotham" and "united", Whereas in new CGUI it is conflicting a lot  
![image](https://user-images.githubusercontent.com/12842642/67864307-6d840b80-fb4b-11e9-9eab-d3f0397b295e.png)

![image](https://user-images.githubusercontent.com/12842642/67864322-74ab1980-fb4b-11e9-8507-cbee38b9f036.png)

Whereas the new CGUI the font-size is conflicting since same is been used for both

    

`<div>

      <h1>heading 1</h1>
      <h2>heading 2</h2>
      <h3>heading 3</h3>
      <h4 className="font-united">heading 4</h4>
      <h5 className="font-united">heading 5</h5>
      <h6 className="font-united">heading 6</h6>
    </div>
    <div className="text-right">
      <h1 className="font-gotham">heading 1</h1>
      <h2 className="font-gotham">heading 2</h2>
      <h3 className="font-gotham">heading 3</h3>
      <h4>heading 4</h4>
      <h5>heading 5</h5>
      <h6>heading 6</h6>
    </div>
`